docs/CHANGELOG.asciidoc

```diff
@@ -82,6 +83,140 @@ Review important information about the {kib} 8.x releases.
 include::upgrade-notes.asciidoc[]
 
 
+[[release-notes-8.17.0]]
+== {kib} 8.17.0
+
+coming::[8.17.0]
+
+For information about the {kib} 8.17.0 release, review the following information.
+
+[float]
+[[deprecations-8.17.0]]
+=== Deprecations
+
+The following functionality is deprecated in 8.17.0, and will be removed in 9.0.0.
+Deprecated functionality does not have an immediate impact on your application, but we strongly recommend
+you make the necessary updates after you upgrade to 8.17.0.
+
+      
+[discrete]
+[[deprecation-197227]]
+.Removes `metrics:allowCheckingForFailedShards` advanced setting.
+[%collapsible]
+====
+*Details* +
+The `metrics:allowCheckingForFailedShards` advanced setting has been removed. With this change, it is no longer possible to suppress warnings about failed shards in TSVB. For more information, refer to ({kibana-pull}197227[#197227]).
+====
```

Comment:
Actually this is not a deprecation, we already removed that functionality in 8.17.
The functionality doesn't break anything, just the expectation of hiding failed shared messages, that is an information that we display anyway in Lens and other visualization editors. It only align the behaviour with everything else

Reply:
We decided to move this under the `Enhancements and bug fixes`